src/html5/node/arena.rs

```diff
@@ -152,239 +152,4 @@ mod tests {
-    //     arena.detach_node(child_id);
-    //     arena.detach_node(parent_id);
-    //
-    //     let parent = arena.get_node(parent_id);
-    //     assert!(parent.is_none());
-    //     let child = arena.get_node(child_id);
-    //     assert!(child.is_none());
-    // }
-    //
-    // #[test]
-    // fn detach_node_with_children_and_parent_and_grandchildren() {
-    //     let mut arena = NodeArena::new();
-    //     let document = Document::shared();
-    //
-    //     let parent = Node::new_element(&document, "parent", HashMap::new(), HTML_NAMESPACE);
-    //     let parent_id = arena.register_node(parent);
-    //     let child = Node::new_element(&document, "child", HashMap::new(), HTML_NAMESPACE);
-    //     let child_id = arena.register_node(child);
-    //     let grandchild = Node::new_element(&document, "grandchild", HashMap::new(), HTML_NAMESPACE);
-    //     let grandchild_id = arena.register_node(grandchild);
-    //
-    //     arena.attach_node(parent_id, child_id, None);
-    //     arena.attach_node(child_id, grandchild_id, None);
-    //     arena.detach_node(child_id);
-    //     arena.detach_node(parent_id);
-    //
-    //     let parent = arena.get_node(parent_id);
-    //     assert!(parent.is_none());
-    //     let child = arena.get_node(child_id);
-    //     assert!(child.is_none());
-    //     let grandchild = arena.get_node(grandchild_id);
-    //     assert!(grandchild.is_none());
-    // }
-    //
-    // #[test]
-    // fn detach_node_with_children_and_parent_and_grandchildren_and_siblings() {
-    //     let mut arena = NodeArena::new();
-    //     let document = Document::shared();
-    //
-    //     let parent = Node::new_element(&document, "parent", HashMap::new(), HTML_NAMESPACE);
-    //     let parent_id = arena.register_node(parent);
-    //     let child = Node::new_element(&document, "child", HashMap::new(), HTML_NAMESPACE);
-    //     let child_id = arena.register_node(child);
-    //     let grandchild = Node::new_element(&document, "grandchild", HashMap::new(), HTML_NAMESPACE);
-    //     let grandchild_id = arena.register_node(grandchild);
-    //     let sibling = Node::new_element(&document, "sibling", HashMap::new(), HTML_NAMESPACE);
-    //     let sibling_id = arena.register_node(sibling);
-    //
-    //     arena.attach_node(parent_id, child_id, None);
-    //     arena.attach_node(child_id, grandchild_id, None);
-    //     arena.attach_node(parent_id, sibling_id, None);
-    //     arena.detach_node(child_id);
-    //     arena.detach_node(parent_id);
-    //
-    //     let parent = arena.get_node(parent_id);
-    //     assert!(parent.is_none());
-    //     let child = arena.get_node(child_id);
-    //     assert!(child.is_none());
-    //     let grandchild = arena.get_node(grandchild_id);
-    //     assert!(grandchild.is_none());
-    //     let sibling = arena.get_node(sibling_id);
-    //     assert!(sibling.is_none());
-    // }
```

Comment:
I'm assuming we won't need these tests, because we already have coverage elsewhere.

Reply:
yes. This is not the place for these tests anymore anyway.